pkg/users/password-dialogs.js

```diff
@@ -137,17 +137,17 @@ function SetPasswordDialogBody({ state, errors, change }) {
             <>
                 <input hidden disabled value={current_user} />
                 <FormGroup label={_("Old password")}
-                           helperTextInvalid={errors && errors.password_old}
-                           validated={(errors && errors.password_old) ? "error" : "default"}
+                           helperTextInvalid={errors?.password_old}
+                           validated={(errors?.password_old) ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18448-20230311-055523-7e3e6fe9-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)